test/jdk/ProblemList.txt

```diff
@@ -651,6 +652,7 @@ javax/sound/sampled/Clip/ClipIsRunningAfterStop.java 8307574 linux-x64
 
 javax/swing/plaf/basic/BasicTextUI/8001470/bug8001470.java 8233177 linux-all,windows-all
 
+javax/swing/JFrame/MaximizeWindowTest.java 8321289 linux-all
```

Comment:
The test was supposed to fail on Linux, but it didn't because of the misreported size. [JDK-8321289](https://bugs.openjdk.org/browse/JDK-8321289)